FirebaseCore/Sources/FIRApp.m

```diff
@@ -46,26 +45,6 @@
 
 #import <objc/runtime.h>
 
-// The kFIRService strings are only here while transitioning CoreDiagnostics from the Analytics
-// pod to a Core dependency. These symbols are not used and should be deleted after the transition.
```

Comment:
git grep -E 'kGGLService|kFIRService' outputs nothing so these weren't used anywhere.

Reply:
This might need a global tap to confirm

Reply:
nvm